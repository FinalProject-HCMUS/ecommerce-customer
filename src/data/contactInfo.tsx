export const contactInfo = {
  phone: '123-456-7890',
  email: 'demo@gmail.com',
  address: '123 Dartmouth Street Boston, Massachusetts 02156 United States',
}
