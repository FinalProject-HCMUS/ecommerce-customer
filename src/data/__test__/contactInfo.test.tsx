import { contactInfo } from '../contactInfo';

describe('contactInfo', () => {
  test('should export an object', () => {
    expect(contactInfo).toBeDefined();
    expect(typeof contactInfo).toBe('object');
  });

  test('should have correct phone, email, and address', () => {
    expect(contactInfo).toHaveProperty('phone', '123-456-7890');
    expect(contactInfo).toHaveProperty('email', 'demo@gmail.com');
    expect(contactInfo).toHaveProperty(
      'address',
      '123 Dartmouth Street Boston, Massachusetts 02156 United States'
    );
  });

  test('should match snapshot', () => {
    expect(contactInfo).toMatchSnapshot();
  });
});