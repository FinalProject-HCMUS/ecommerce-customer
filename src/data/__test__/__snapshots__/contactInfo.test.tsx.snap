// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contactInfo should match snapshot 1`] = `
{
  "address": "123 Dartmouth Street Boston, Massachusetts 02156 United States",
  "email": "demo@gmail.com",
  "phone": "123-456-7890",
}
`;
